src/docs/compact.dox

```diff
@@ -2,11 +2,50 @@
 
 The WT_SESSION::compact method can be used to compact data sources.
 
-The data source does not have to be quiescent, compaction may be
-performed on an active data source.
+The data source does not have to be quiescent, compaction may be performed on an active data source.
```

Comment:
What does this mean here? Maybe it's worth a rewrite

Reply:
I rewrote that section here 537812b6b3d07314f48896adc7cab31be3390a38, let me know what you think.